core/web/resolver/query.go

```diff
@@ -230,3 +232,36 @@ func (r *Resolver) Node(ctx context.Context, args struct{ ID graphql.ID }) (*Nod
 
 	return NewNodePayloadResolver(&node, nil), nil
 }
+
+// VRFKeys fetches all VRF keys.
+func (r *Resolver) VRFKeys(ctx context.Context) (*VRFKeysPayloadResolver, error) {
+	if err := authenticateUser(ctx); err != nil {
+		return nil, err
+	}
+
+	keys, err := r.App.GetKeyStore().VRF().GetAll()
+	if err != nil {
+		return nil, err
+	}
+
+	return NewVRFKeysPayloadResolver(keys), nil
+}
+
+// VRFKey fetches the VRF key with the given ID.
+func (r *Resolver) VRFKey(ctx context.Context, args struct {
+	ID graphql.ID
+}) (*VRFKeyPayloadResolver, error) {
+	if err := authenticateUser(ctx); err != nil {
+		return nil, err
+	}
+
+	key, err := r.App.GetKeyStore().VRF().Get(string(args.ID))
```

Comment:
Handle `ErrLocked` error case.

Reply:
I've already addressed this in #5376, but this will be resolved in future PRs. See this comment for deets: https://github.com/smartcontractkit/chainlink/pull/5376#issuecomment-964322507.